src/compiler/emitter.ts

```diff
@@ -4219,6 +4219,9 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, Promi
                 scopeEmitStart(node, "constructor");
                 increaseIndent();
                 if (ctor) {
+                    // Emit all the directive prologues (like "use strict").  These have to come before
+                    // any other preamble code we write (like parameter initializers).
+                    var startIndex = emitDirectivePrologues(ctor.body.statements, /*startWithNewLine*/ true);                    

```

Comment:
Please define startIndex outside the if statement, and initialize it appropriately.